proto/kava/incentive/v1beta1/claims.proto

```diff
@@ -12,6 +12,8 @@ option (gogoproto.goproto_getters_all) = false;
 
 // BaseClaim is a claim with a single reward coin types
 message BaseClaim {
+  option (cosmos_proto.implements_interface) = "Claim";
```

Comment:
"Claim" was added here but taken away in line 328 of proto-docs.md?

Reply:
Claim in proto-docs corresponds to the claim message at the bottom of this file

I don't believe this line was really doing anything before since there wasn't a Claim interface either.